app/views/articles/search.html.erb

```diff
@@ -9,6 +9,14 @@
      data-articles-since="<%= Timeframer.new(params[:timeframe]).datetime&.iso8601 %>">
   <%= render "articles/search/sidebar" %>
   <div class="articles-list" id="articles-list">
+    <div class="flex-1 flex items-center justify-between">
+      <h2 class="fs-l fw-heavy">Results</h2>
+      <nav id="sorting-option-tabs" class="crayons-tabs hidden s:flex">
+        <a href="javascript:;" class="crayons-tabs__item <%= "crayons-tabs__item--current" if @current_ordering == :relevance %>">Most Relevant</a>
```

Comment:
I know `href="javascript;"` is kind of weird. I intend to come back to this later and change it but didn't have time to get to that point this morning.